packages/sdk/src/sync-agent/utils/spaceUtils.ts

```diff
@@ -26,7 +26,7 @@ export async function makeDefaultMembershipInfo(
             duration: 0,
             currency: ETH_ADDRESS,
             feeRecipient: feeRecipient,
-            freeAllocation: 0,
+            freeAllocation: 1000,
```

Comment:
@texuf I'm unsure of the implications of this change. Does this break any expectations for the sync agent running in production? This method creates a default membership info.

Reply:
this is fine. We need to consolidate this code.